plugins/dependency-graph-resolver/src/main/java/eu/f4sten/depgraph/endpoints/Hello.java

```diff
@@ -0,0 +1,63 @@
+package eu.f4sten.depgraph.endpoints;
+
+import java.util.Set;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import eu.f4sten.depgraph.data.Coordinates;
+import eu.f4sten.depgraph.data.Naming;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+@Path("/hello")
```

Comment:
should we keep this example class in the depgraph.endpoints package?

Reply:
Yes, the endpoints here are for pure illustration/testing purposes... as soon as I will migrate the *actual* depgraph code to the new repo, I will get rid of these endpoints.